lib/strategies/base.js

```diff
@@ -80,6 +80,10 @@ BaseStrategy.prototype._reformatInput = function () {
 
   input.value = formattedState.value;
   setSelection(input, formattedState.selection.start, formattedState.selection.end);
+
+  if (this._afterReformatInput) {
```

Comment:
Sorry for the confusing explanation. My idea was that you could remove the `if (this._afterReformatInput)` check if it was defined in BaseStrategy as a no-op method that would be overridden by other strategies, i.e. AndroidChromeStrategy. Feel free to ignore 👍